app/models/stop.rb

```diff
@@ -284,6 +291,52 @@ def clean_attributes
   end
 end
 
+class StopStation < Stop
+  current_tracked_by_changeset({
+    kind_of_model_tracked: :onestop_entity,
+    virtual_attributes: [
+      :served_by,
+      :not_served_by,
+      :identified_by,
+      :not_identified_by,
+      :imported_from_feed
+    ]
+  })
+  # Station relations
+  has_many :stop_entrances, class_name: 'StopEntrance', foreign_key: :parent_stop_id
+  has_many :stop_platforms, class_name: 'StopPlatform', foreign_key: :parent_stop_id
+end
+
+class StopPlatform < Stop
+  current_tracked_by_changeset({
+    kind_of_model_tracked: :onestop_entity,
+    virtual_attributes: [
+      :served_by,
+      :not_served_by,
+      :identified_by,
+      :not_identified_by,
+      :imported_from_feed,
+      :parent_stop_onestop_id
+    ]
+  })
+  belongs_to :parent_stop, class_name: 'Stop'
+end
+
+class StopEntrance < Stop
+  current_tracked_by_changeset({
+    kind_of_model_tracked: :onestop_entity,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.